west.yml

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+# West YAML manifest for Zephyr as Xen Dom0
+
+manifest:
+  self:
+    path: zephyr-optee-test
+
+  remotes:
+    - name: xentroops
+      url-base: https://github.com/xen-troops
+
+    - name: zephyr
+      url-base: https://github.com/zephyrproject-rtos
+
+  projects:
+    - name: zephyr
+      remote: xentroops
+      revision: zephyr-v3.3.0-xt
+      west-commands: scripts/west-commands.yml
+
+    - name: zephyr-xenlib
+      remote: xentroops
+      revision: main
+
+    - name: zephyr-optee-client
+      remote: xentroops
+      revision: master
+
+    - name: littlefs
```

Comment:
Do we use littlefs or zephyr-xenlib in this application?

Reply:
We definitely use littlefs for secure storage. As for the zehyr-xenlib - I don't know. You've added it :). Probably we don't need but I'll recheck and update

Reply:
updated